semantic_model_generator/generate_model.py

```diff
@@ -21,6 +21,7 @@
 
 _PLACEHOLDER_COMMENT = "  "
 _FILL_OUT_TOKEN = " # <FILL-OUT>"
+_DEFAULT_N_SAMPLE_VALUES_PER_COL = 10
```

Comment:
can we make it to 3?
Or to your earlier point have it be 3 for non-string type and only 10 for string type?

Reply:
Let's do that as a fast follow. 

I think we should leave as 10 and let clients update? Especially with retrieval launching, more should be better.